9.1 Specifications page updates

kwsutter opened the pull request:
Signed-off-by: Kevin Sutter <kwsutter@gmail.com>

## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [ ] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [ ] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
N/A
- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [x] Updated release record
   - [ ] Generated IP Log
   - [ ] Email to PMC
   - [ ] Start release review by emailing EMO 
- [ ] The URL of the OSSRH staging repository for the api, javadoc:
      <add URL here>
- [x] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/promoted/jakarta-jakartaeetck-9.1.0.zip
- [ ] The URL of the compatibility certification request issue(s):
      <add URL here>
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory. 

Note: If any item does not apply, check it and mark N/A below it.

*NOTE:* I am also including an update to the 9.0 pages for the multiple TCKs that have been produced since the 9.0.0 TCK.

kwsutter commented:
Still in Draft form...  Waiting for a few minor changes on the Platform Spec and API pom to be completed before generating the final versions.  Tomitribe (@dblevins and @jeanouii) were asked to be mentors of this effort.  I'll take comments at any time (even before I take it out of Draft mode).  Thanks!

kwsutter commented:
Took this out of Draft mode.  It's pretty much ready for review.  Still waiting on Open Liberty and Wildfly for their CCRs, but otherwise review away...  Due to the huge number of javadoc files, I'd suggest using the https://deploy-preview-372--jakartaee-specifications.netlify.app/ preview.  Thanks!

gerdogdu commented:
@kwsutter I have also added our  certification request via https://github.com/eclipse-ee4j/jakartaee-platform/issues/350
I will be updating our test results pages, some couple of tests are remaining in the run...
Thanks.
Gurkan

gerdogdu commented:
@kwsutter We have finished to run tests and updated our Test Results page.

kwsutter commented:
> @kwsutter I have also added our certification request via [eclipse-ee4j/jakartaee-platform#350](https://github.com/eclipse-ee4j/jakartaee-platform/issues/350)
> I will be updating our test results pages, some couple of tests are remaining in the run...
> Thanks.
> Gurkan

Hi, @gerdogdu.  Per my comments on your CCR (https://github.com/eclipse-ee4j/jakartaee-platform/issues/350), I don't see a need to repeat the designation of a Glassfish derivative as a CI used for ratification.  As a reminder, the CI's used for the ballot are only for demonstrating that the Specification is implementable and testable.  Eclipse Glassfish has already provided that capability.  Unless ManagedCat is modifying or adding new features to Glassfish, there really is no reason to include it on the ballot since there is nothing "new" with this derivative.  Make sense?

But, please do follow through on the process to get ManagedCat listed on the Compatible Products page!

gerdogdu commented:
Hi Kevin,
I have replied to [eclipse-ee4j/jakartaee-platform#350]( https://github.com/eclipse-ee4j/jakartaee-platform/issues/350)

jeanouii commented:
@kwsutter ManageFish PR got approved just on time.
If you can add it to this PR with the links, I'm ready to send out the ballot email.

edbratt commented:
For consistency, would it be appropriate to point at the Specific WildFly .zip downloads?

jeanouii commented:
@edbratt I think so. @starksm64 thoughts?

jeanouii reviewed (approved):
Looks good to me.
Please don't update the PR from now on as the vote will go off for ballot and we should not change the PR while the vote is up.

kwsutter commented:
> Looks good to me.
> Please don't update the PR from now on as the vote will go off for ballot and we should not change the PR while the vote is up.

@jeanouii Your previous comments suggested an update to include ManagedCat.  And, with @edbratt's comment, you thought it was a good idea to include a specific link to the Wildfly download.  But, then I saw this comment that said the PR is frozen because the vote had been started...  

I see that @ivargrimstad has posted a similar concern on the Ballot thread.  I think these updates should be performed and whether the vote needs to be restarted or not is something we can discuss at tomorrow's Spec Committee call.  Since nobody has formally voted yet, we may be okay...  

Thanks!

kwsutter commented:
All,
A couple of minor updates to answer the questions previously posted...
- Removed the errant entry for TomEE at the Platform level
- Added an entry for ManageFish 6.1 at the Platform level
- Modified the links for the WildFly download to be more specific

We will be discussing the ManageFish CI request and the impact of these changes to the active ballot at tomorrow's Spec Committee call.  To that end, these may not be the final changes...  :-)

jeanouii reviewed (approved):
Perfect. 
As discussed, PR updates look good and match our discussion.
No one voted yet so we are good

jeanouii commented:
Ballot completed today on time
https://www.eclipse.org/lists/jakarta.ee-spec/msg01722.html

Ready to go. Thank you Kevin and everyone

dblevins commented:
Couple minor updates if we move forward with this format.  ASF infra reminded us we need to use the mirrors link, which would be:

 - https://www.apache.org/dyn/closer.cgi/tomee/tomee-9.0.0-M7/apache-tomee-9.0.0-M7-plume.zip

Also, we need to ensure "TomEE" is updated to "Apache TomEE"

gerdogdu commented:
Our ManageFish download link has also changed to below link and we also updated the link in our [compatibility request issue](https://github.com/eclipse-ee4j/jakartaee-platform/issues/350)

- https://download.managecat.com/6.1.0/managefish-fullprofile-6.1.0.zip

Regards.

jeanouii commented:
- [X] adds this final checklist to the main PR.
 - [X] adds the `approved` label to the PRs, and sends out the Ballot Summary per this [template](#ballot-summary-email-template) to the [public Jakarta EE Specification Committee email list](jakarta.ee-spec@eclipse.org)
 - [] calculates the staged EFTL TCK signature and promotes it to the committee download area
  using the https://ci.eclipse.org/jakartaee-spec-committee/job/promote-release/ job. Manually editing the jenkins Build Information will help identify the build (ie. Mail 2.0 or CDI 3.0).
 - [] merges the specification (and apidocs) PRs, ensuring the "date:" field in the _index.md file has an appropriate value to allow publishing.
 - [] updates the specification page with the ballot results. This is normally done via a separate PR that should be reviewed, approved, and merged.
 - [] notifies the EMO of the ballot results by email to [emo@eclipse-foundation.org](emo@eclipse-foundation.org). Just forward the ballot summary note sent earlier to the public Spec Committee email list.
 - [X] creates an issue in the specification project that includes the following checklist for the specification project team:
https://github.com/eclipse-ee4j/jakartaee-platform/issues/361

jeanouii commented:
@kwsutter added the checklist to conclude the ballot. Can you do the 2 small changes from David and Gurkan?

kwsutter commented:
> @kwsutter added the checklist to conclude the ballot. Can you do the 2 small changes from David and Gurkan?

Yes, doing that now.  Thanks!
